test/exporter_test.rb

```diff
@@ -33,6 +33,7 @@ def test_all_functions # break this into separate parts
     image = {
       height: 20,
       width: 20,
+      id: 1,
```

Comment:
I think we needed this; we may need to refactor so it auto-adds the image ids here based on the order they're brought in, instead of relying on an external unique ID